src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -18,6 +18,7 @@ public InstructorFeedbacksPageData(AccountAttributes account) {
     }
 
     public String courseIdForNewSession;
+    public String feedbackSessionNameForNewSession;
```

Comment:
The variable name should be something different from courseIdForNewSession, since courseIdForNewSession is used to set the default course id for the "Add New Feedback Session" form, and this new variable doesn't.